shared/versions/stack/8.18.asciidoc

```diff
@@ -0,0 +1,77 @@
+:version:                8.18.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           8.18.0
+:logstash_version:       8.18.0
+:elasticsearch_version:  8.18.0
+:kibana_version:         8.18.0
+:apm_server_version:     8.18.0
+// branch is 8.18 instead of 8.x for long-term linking purposes
+:branch:                 8.18
+:minor-version:          8.18
+:major-version:          8.x
+:prev-major-version:     7.x
+:prev-major-last:        7.17
+:major-version-only:     8
+:ecs_version:            8.18
```

Comment:
@lcawl , @bmorelli25 : would that line on ecs_version be all right? (this is related with my previous question on conf.yaml about ECS)

Reply:
It should be left unchanged per earlier reply about ECS doing its own thing.

Reply:
`8.18.asciidoc` is a new file. What do you mean with `unchanged` in this context?

Reply:
Ok, I see what you mean, sorry :) the example provided in the template left ecs in a previous version in the new stack version file, so I'm doing the same. Fixed it... thanks!